rkt/rkt.go

```diff
@@ -181,7 +183,9 @@ func init() {
 
 func getTabOutWithWriter(writer io.Writer) *tabwriter.Writer {
 	aTabOut := new(tabwriter.Writer)
+
```

Comment:
I must be confusing it with another guide, but using blank spaces to logically grouping statements improves code quality. From the functions I've read, there are a lot of clustered statements. I believe blank spaces would help with readability and blank spaces are cheap :)